### PR TITLE
Add paging through fetched playlists

### PR DESCRIPTION
**Pull request by adamfitzgibbon**
- addresses #44 
- uses the offset parameter to page through additional playlists, should the user have more than 50 followed playlists
- this ended up being a bit more complex than I would have liked it, so it ended up being a little messy. Feel free to comment with ways to clean these changes up!

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -35,31 +35,76 @@ def str2bool(self, v):
...
+            print("You don't have any Spotify playlists!")
+            return
+        self.printOwnedPlaylists(ownedPlaylists)
         enteringPlaylists = True
         playlistsToAdd = []
         playlistsToAddIndices = []
         while enteringPlaylists:
-            index = input('Enter the number of the playlist you would like to add: ').strip()
+            print()
+            userInput = input('Enter the number of the playlist you would like to add, or [N/B] to fetch more/previous playlists: ').strip()
```
Comment (amcquade) [inline models.py]: there should be an option to quit/exit this loop.

**Comment by adamfitzgibbon** [comment]
Hey sorry I got a bit busy but I'll still address this possibly this weekend. I'm only displaying playlists that you own. Are you testing with your own playlists or playlists that you follow?

**Comment by adamfitzgibbon** [comment]
Also, I didn't realize that you added a GUI. Sweet! I didn't have time to get flask setup tonight but I'll do that as well so I can test with that as well.

**Comment by amcquade** [comment]
@adamfitzgibbon sounds good, ill try to test that code this weekend. let me know if you have any issues setting up flask

**Pull request merged**